docs/data/base/components/button/UnstyledButtonIntroduction/css/index.js

```diff
@@ -54,42 +56,62 @@ function Styles() {
 
   return (
     <style>{`
-  .IntroductionButton {
-    font-family: 'IBM Plex Sans', sans-serif;
-    font-weight: 600;
-    font-size: 0.875rem;
-    line-height: 1.5;
-    background-color: ${cyan[500]};
-    padding: 8px 16px;
-    border-radius: 8px;
-    color: white;
-    transition: all 150ms ease;
-    cursor: pointer;
```

Comment:
Should we start the `cursor: pointer` discussion? 😄  I am just joking, I would love to follow the default browser behavior, however just noting that this is opposite to what we use in Material UI, which kind of feels better 😄

Reply:
> Should we start the cursor: pointer discussion?

I think we just did lol

> however just noting that this is opposite to what we use in Material UI

imo buttons should not have cursor: pointer, but it's not a hill I want to die on haha. So you choose, just say the word and I will add it?